R/visualization.R

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
does this function need an roxygen docstring?

Reply:
This function is for a different PR. I removed it from this one. Good spot!